test/delete_channel.js

```diff
@@ -118,7 +118,7 @@ describe('Channels - Delete', function() {
 	});
 
 	it('Recover a deleted channel', async function() {
-		const [client, _, oldMessage, deleteResponse] = await setupDeletedChannel();
+		const [client, , oldMessage] = await setupDeletedChannel();
```

Comment:
`, ,`??

Reply:
Same issue at multiple places in this PR.

Reply:
It's the idiomatic way of writing: don't write unless needed.

Otherwise, for example, lodash could be shadowed.

see https://www.freecodecamp.org/news/array-destructuring-in-es6-30e398f21d10/#ignoring-some-values for more examples